File: CRM/src/app/shared/eHospital_DongNai_A/BenhAnTongQuat_LienQuanBenh.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { BenhAnTongQuat_LienQuanBenh } from './BenhAnTongQuat_LienQuanBenh.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class BenhAnTongQuat_LienQuanBenhService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'BenhAnTongQuat_LienQuanBenh_Id', 'BenhAnTongQuat_Id', 'NoiDung_Id', 'KyHieu', 'ThoiGian'];

    List: BenhAnTongQuat_LienQuanBenh[] | undefined;
    ListFilter: BenhAnTongQuat_LienQuanBenh[] | undefined;
    FormData!: BenhAnTongQuat_LienQuanBenh;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "BenhAnTongQuat_LienQuanBenh";
    }
}

File: Service_eHospital_DongNai_A/Implement/BenhAnTongQuat_LienQuanBenhService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class BenhAnTongQuat_LienQuanBenhService : BaseService<BenhAnTongQuat_LienQuanBenh, IBenhAnTongQuat_LienQuanBenhRepository>
    , IBenhAnTongQuat_LienQuanBenhService
    {
    private readonly IBenhAnTongQuat_LienQuanBenhRepository _BenhAnTongQuat_LienQuanBenhRepository;
    public BenhAnTongQuat_LienQuanBenhService(IBenhAnTongQuat_LienQuanBenhRepository BenhAnTongQuat_LienQuanBenhRepository) : base(BenhAnTongQuat_LienQuanBenhRepository)
    {
    _BenhAnTongQuat_LienQuanBenhRepository = BenhAnTongQuat_LienQuanBenhRepository;
    }
    }
    }

